src/main/java/duke/fileHandler/FileHandler.java

```diff
@@ -0,0 +1,69 @@
+package duke.fileHandler;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class FileHandler {
+    private static String filePath;
+
+    public static void setfilePath(String filePath) {
+        FileHandler.filePath= filePath;
+    }
+
+    public static Task stringToTask(String input) {
+        int indexOfSpace = input.indexOf(" ");
+        String taskType = input.substring(indexOfSpace + 2, indexOfSpace+3);
+        String status = input.substring(indexOfSpace + 5, indexOfSpace + 6);
+        String nameAndDate = input.substring(indexOfSpace+ 7);
```

Comment:
There are many [magic numbers](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers) here which you can avoid.